pkg/model/manifest.go

```diff
@@ -368,12 +368,15 @@ func ToRuns(cmds []Cmd) []Run {
 }
 
 type PortForward struct {
-	// The port to expose on localhost of the current machine.
-	LocalPort int
-
 	// The port to connect to inside the deployed container.
 	// If 0, we will connect to the first containerPort.
 	ContainerPort int
+
+	// The port to expose on the current machine.
+	LocalPort int
+
+	// Optional host to bind to on the current machine (localhost by default)
+	Host *string
```

Comment:
what's the purpose of making it a pointer? would we want to treat the `nil` value differently than the empty string?

Reply:
Two thoughts:
A) Even if a piece of information is not currently used, but might be used in the future, and capturing it doesn't require any overhead, imo it's better to capture it now than to refactor later.
B) Right now if the value is `nil` we do port forwarding without addresses, and if the value is empty string we simply pass it to `NewOnAddress`, which I'm assuming explicitly fails with some error (tho I haven't tested it tbh). So this signature makes it slightly easier to catch bugs in case hostname is meant to be supplied, but ended up being empty string. This is probably rather opinionated.